benefits/untracked_holiday.md

```diff
@@ -4,6 +4,10 @@ We're building a high-performance organisation that delivers great software prod
 
 We have adopted an untracked holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. 
 
+It is important that you and your colleagues feel as though that holiday is highly available. That holiday is an easily accessible commodity that enables everyone to get the necessary rest that they need. We intend untracked holiday to prevent burn-out in situations, where individuals feel that they need to "save up" holiday days, or plan elaborate strategies around using your holiday allowance "effectively" later in the year, rather than taking holiday when needed most.
```

Comment:
feel that holiday
OR 
feel as though holiday

(probably the former for my taste)